ScoutSuite/providers/aws/provider.py

```diff
@@ -183,14 +180,14 @@ def add_security_group_name_to_ec2_grants_callback(self, current_config, path, c
                 target.append(sg_id)
             else:
                 target = current_path[:(
-                        current_path.index('security_groups') + 1)]
+                    current_path.index('security_groups') + 1)]
                 target.append(sg_id)
             ec2_grant['GroupName'] = get_value_at(
                 self.services['ec2'], target, 'name')
 
     @staticmethod
     def _process_cloudtrail_trails(cloudtrail_config):
-        print_info('Processing CloudTrail config')
+        print_info('Processing CloudTrail config...')
```

Comment:
The `...` were removed.
```suggestion
        print_info('Processing CloudTrail config')
```